workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -23,6 +26,22 @@ public EntityGameObjectLinker(World world, WorkerSystem worker)
             entityManager = world.GetExistingManager<EntityManager>();
         }
 
+        /// <summary>
+        ///     Links a GameObject to an ECS Entity.
+        /// </summary>
+        /// <remarks>
+        ///     All <see cref="UnityEngine.Component"/>s on the GameObject will be inserted onto the ECS Entity.
+        ///     A <see cref="SpatialOSComponent"/> will be added to the GameObject.
+        /// </remarks>
+        /// <remarks>
+        ///     If a Monobehaviour exists multiple times on the GameObject, only the first occurence will be inserted
+        ///     onto the ECS entity.
+        /// </remarks>
+        /// <param name="gameObject">The GameObject to link.</param>
+        /// <param name="entity">The entity to link.</param>
+        /// <param name="viewCommandBuffer">
+        ///     An instance of the ViewCommandBuffer. Should be flushed after this method returns.
```

Comment:
would change the "should" to a "must" -> if it's not flushed, the whole linking won't work

Reply:
Must be flushed for changes to take effect